doc/source/examples/03_modeling/scale_map_bodies.mystnb

```diff
@@ -0,0 +1,233 @@
+   - A new design sketch named "asymmetric_cube" is created.
+
+```{code-cell} ipython3
+# Initialize the sketch design
+design = m.create_design("asymmetric_cube")
+```
+
+2. **Create Asymmetric Sketch Profile:**
+   - We will make a sketch profile which is basically a cube centered on the origin with side length 2 with a cutout.
+
+```{code-cell} ipython3
+# Create the cube profile with a cut through it
+asymmetric_profile = Sketch().segment(Point2D([1, 1]), Point2D([-1, 1])).segment_to_point(Point2D(
+    [0, 0.5])).segment_to_point(Point2D([-1, -1])).segment_to_point(Point2D([1, -1])).segment_to_point(Point2D([1, 1]))
+```
+
+3. **Create Asymmetric Body:**
+   - `extrude_sketch` on the `asymmetric_profile`, as the base sketch, creating the 3D cube, with a cutout, with `distance` being 1.
+
+```{code-cell} ipython3
+# Extrude the asymmetric profile by distance of 1 unit
+body = design.extrude_sketch("box", asymmetric_profile, 1)
+
+design.plot()
+```
+
+4. **Apply Map Reframing:**
+   - We will first make a copy of the shape and translate it in 3D space so that we can view them side by side, then we will apply the reframing to the copied shape.
+   - Note: The below map uses the default x direction, but the y direction is swapped with the z direction, effectively rotating the original shape so that it is standing vertically.
+
+```{code-cell} ipython3
+copied_body = body.copy(body.parent_component, "copied_body")
+# Apply the reframing
+copied_body.map(Frame(Point3D([0, 0, 0]), UnitVector3D([1, 0, 0]), UnitVector3D([0, 0, 1])))
+# Shift the new modified body in the plane in the negative y direction by 2 units
+copied_body.translate(UnitVector3D([0, 1, 0]), -2)
+
+design.plot()
+```
+
+## Mirror Body
+
+The `mirror` function is designed to mirror the geometry of a body across a specified plane. This function plays a role in geometric transformations, enabling the reflection of bodies to create symmetrical designs.
+
+### Usage
+
+To use the `mirror` function, you call it on an instance of a geometry body, passing a single argument: the plane across which the body will be mirrored. This plane is represented by a `Plane` object, defining the axis of symmetry for the mirroring operation.
+
+```python
+body.mirror(plane)
+```
+
++++
+
+## Example: Traignular Body
+The code snippets below demonstrates how to reframe a cube body using `mirror` function in `body.py`. The process involves initializing a sketch design for the body profile, extruding the profile by a distance, and then performing the mirroring operation to reflect the shape over the specified axis.
+1. **Initialize the Shape Sketch Design:**
+   - A new design sketch named "traignle" is created.
+
+```{code-cell} ipython3
+# Initialize the sketch design
+design = m.create_design("triangle")
+```
+
+2. **Define Parameters:**
+   - `point1` the first vertex of the triangle.
+   - `point2` the second vertex of the triangle.
+   - `point3` the third vertex of the triangle.
```

Comment:
```suggestion
2. **Define parameters:**
   - `point1`: First vertex of the triangle
   - `point2`: Second vertex of the triangle
   - `point3`: Third vertex of the triangle.
```
I don't know if you need a blank line before starting a list like you do in a reSTructuredText file.